src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, val) {
+      callback.call(this, val);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(val) {
+      this.result += val;
+
+      return this;
```

Comment:
you return this inside operate method, so this makes the chain call possible itself

Reply:
Fixed 😅